src/orbit-db-address.js

```diff
@@ -15,51 +15,32 @@ class OrbitDBAddress {
   }
 
   static isValid (address) {
```

Comment:
I believe this function is at the top of the execution stack and can fail in two ways, as per the tests

1. If the CID is invalid
2. If the address is missing a prefix.

Instead of returning `false` here and the throwing the error in `parse`, what about throwing specific errors here. That would make things more explicit i.e.

1. "Invalid OrbitDB address: xxxx is not a valid CID"
2. "Invalid OrbitDB address: missing /orbitdb/ prefix"

Reply:
I dont think isValid should result in a boolean or an error. Having these two, more specific, errors is a good idea though.

This api is going to be changed soon to have similar methods to cids so I think we keep it the same for now and do this with https://github.com/tabcat/SCPs/blob/SCP-11/scps/SCP-11.md#address-class 's asAddress method.